packages/core/src/mask/MaskSystem.js

```diff
@@ -80,7 +80,7 @@ export default class MaskSystem extends System
         }
         else if (this.enableScissor
             && !this.scissor
-            && this.renderer._activeRenderTarget.root
+            // && this.renderer._activeRenderTarget.root
```

Comment:
we should replace this line with: 
```&& !this.renderer.renderTexture.current```
this makes sure we only use the stencil mask if we are on the root.

Reply:
👍